src/containers/EventCreationPage.tsx

```diff
@@ -28,6 +28,12 @@ const EventCreationPage = () => {
         data: { events: events.concat([addEvent]) },
       });
     },
+    onCompleted({ addEvent }) {
+      history.replace('/events', { addedEventId: addEvent.id });
+    },
+    onError() {
+      history.replace('/events');
```

Comment:
We might want more to be down `onError`, but for now I included the "default" behaviour which just navigates the user to the `/events` page without setting an `addedEventId` value in the location state